pkg/kratos/handlers_test.go

```diff
@@ -293,15 +293,15 @@ func TestHandleCreateFlowRedirectToSetupWebauthn(t *testing.T) {
 		t.Errorf("expected error to be nil got %v", err)
 	}
 
-	if res.StatusCode != http.StatusSeeOther {
+	if res.StatusCode != http.StatusOK {
 		t.Fatalf("expected HTTP status code 303 got %v", res.StatusCode)
```

Comment:
```suggestion
		t.Fatalf("expected HTTP status code 200 got %v", res.StatusCode)
```

same for https://github.com/canonical/identity-platform-login-ui/blob/af13afdceaf73850c086e0d0f26762e62fa41694/pkg/kratos/handlers_test.go#L658

Reply:
fixed, thank you